### ISSUE TITLE
Crashing on device with XCode 15 archives

### ISSUE THREAD

**Issue by JanHalozan** [twilio/conversations-ios#48]
I've got an issue with the Twilio conversations SDK. Generally I'm using it via SPM but I've also tried a manual .xcframework install. In both cases the app work on the simulator, runs on my iPhone connected via Xcode but crashes when I archive it and submit it to TestFlight.

In the TF crash report I can see the following:

```
Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Termination Reason: DYLD 1 Library missing
Library not loaded: @rpath/TwilioConversationsClient.framework/TwilioConversationsClient
```

So it looks like that something is causing the framework to not be loaded. I've checked all of the usual things:
- Clean and build, reopen Xcode for starters
- Making sure ObjC and stuff is set in Other Linker Flags
- Removing the SDK from SPM and adding it again
- Upgrading to the latest version (didn't work so I downgraded to 2.2.8)
- Removing the SPM build and installing via .xcframework

None of the solutions work for me. I was using XCode 14 before and it worked well. I've updated to Sonoma and Xcode 15 now so a downgrade to XCode 14 isn't possible anymore.

Any thoughts on what could be causing the issue?

**Comment by berkus** [comment]
Looks like a frameworks copy step is missing from your build steps.

Did you try steps from the [framework readme](https://github.com/twilio/conversations-ios/blob/master/README.md#swift-package-manager)?

Which version of the SDK are you trying to integrate?

**Comment by JanHalozan** [comment]
Yes I've got the exact same setup as in the instructions. I tried integrating via both SPM and Manually - downloading the framework. I'm using 2.2.8 however I also tried updating to 4.x without any success. It works when I run the app in the simulator and it also works if I build and run on my iPhone via Xcode.

**Comment by berkus** [comment]
There is something missing from the archive step, most likely the framework file is not copied in the archive.

**Comment by JanHalozan** [comment]
It indeed looks like to be the issue. However exactly the same process works fine on my colleague's Xcode 14. I tried removing and following the instructions to the letter and it doesn't link with Xcode 15. Have there been any other reports of such issues?
